File: hub-kit-backend/src/main/java/com/inetum/realdolmen/hubkitbackend/dto/DriverDTO.java
package com.inetum.realdolmen.hubkitbackend.dto;

import lombok.*;
import org.antlr.v4.runtime.misc.NotNull;

import java.time.LocalDate;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class DriverDTO {
    private Integer id;
    @NotNull
    private String firstName;
    @NotNull
    private String lastName;
    @NotNull
    private LocalDate birthday;
    @NotNull
    private String address;
    @NotNull
    private String country;
    private String phoneNumber;
    private String email;
    @NotNull
    private String drivingLicenseNr;
    @NotNull
    private String category;
    @NotNull
    private LocalDate drivingLicenseExpirationDate;
}

File: crash-kit-app/app/src/main/java/com/inetum/realdolmen/crashkit/dto/DriverDTO.kt
package com.inetum.realdolmen.crashkit.dto

import java.time.LocalDate

data class DriverDTO(
    var firstName: String? = null,
    var lastName: String? = null,
    var birthday: String? = null,
    var address: String? = null,
    var country: String? = null,
    var phoneNumber: String? = null,
    var email: String? = null,
    var drivingLicenseNr: String? = null,
    var category: String? = null,
    var drivingLicenseExpirationDate: String? = null
)

